presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -828,6 +828,17 @@ Optimizer Properties
     Enable analysis and propagation of logical properties like distinct keys or cardinality among the nodes of
     a query plan. The optimizer may then use these properties to perform various optimizations.
 
+``optimizer.use-histograms``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default Value:** ``false``
+
+    Enables the optimizer uses histograms when available to perform the cost estimate calculations
+    during query optimization. This parameter does not prevent histograms from being collected by
```

Comment:
```suggestion
    during query optimization. When set to ``false``, this parameter does not prevent histograms from being collected by
```
Suggestion to make clear what setting does what. If I've misunderstood the behavior, please correct this.

Reply:
fixed